README.md

```diff
@@ -98,149 +98,142 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Fixed broken links in doc/devdocs/readme.md. Thanks [@jem-experience](https://github.com/jem-experience)!
 
 ### Development
 
- - Fixed dependency issues on upgrading .NET from 8.0.0 to 8.0.1.
- - Upgraded Microsoft.Extensions.ObjectPool from .NET 5 to .NET 8.
- - Upgraded the Windows SDK Build Tools to 10.0.22621.2428.
- - Upgraded the Windows Implementation Library to 1.0.231216.1.
- - Upgraded NLog.Schema to 5.2.8 and NLog.Extensions.Logging to 5.3.8.
- - Upgraded Markdig.Signed to 0.34.0.
- - Upgraded Microsoft.NET.Test.Sdk to 17.8.
- - Upgraded CommunityToolkit.WinUI dependencies to 8.0.240109.
- - Upgraded CommunityToolkit.Mvvm to 8.2.2. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Upgraded Windows App SDK to 1.4.4. Thanks [@snickler](https://github.com/snickler)!
- - Upgraded WPFUI version to 3.0.0-preview.13. Thanks [@niels9001](https://github.com/niels9001)!
- - Upgraded StyleCop.Analyzers to 1.2.0-beta.556. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Upgraded Microsoft.Windows.Compatibility to 8.0.1.
- - Upgraded System.Data.SqlClient to 4.8.6.
- - Consolidate XAML Namespaces across the solutions. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
- - Removed the toolkit labs package source reference, since the controls we were using made it to the generally available community toolkit.
- - Added Microsoft.MSBuildCache to experiment with build caching to reduce pipeline runs duration. Thanks [@dfederm](https://github.com/dfederm)!
- - Configured the release CI to follow the latest 1ES pipeline release version again.
- - Removed the copyright year from assembly information. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Added the Command Not Found entry to the GitHub templates.
- - Removed unused code for a GPO policy to control auto updating of PowerToys. Thanks [@htcfreek](https://github.com/htcfreek)!
- - Improved code behind for getting the localization of context menu entries.
- - Locked some terms in resource files to avoid localization.
-
-#### What is being planned for version 0.79
-
-For [v0.79][github-next-release-work], we'll work on the items below:
+ - Updated Microsoft.MSBuildCache to 0.1.258-preview. Thanks [@dfederm](https://github.com/dfederm)!
+ - Fixed CI to point VCToolsVersion to VC.CRT instead of VC.Redist version. Thanks [@snickler](https://github.com/snickler)!
+ - Updated MSTest adapter and framework to 3.2.
+ - Fixed CI by pointing WiX 3.14 urls and hashes to the latest release on GitHub.
+ - Added Pro and Enterprise editions of Visual Studio to the repository's development configuration DSC scripts.
+ - Updated CppWinRT to 2.0.240111.5.
+ - Updated System.Drawing.Common to 8.0.2 to fix CI builds after the .NET 8.0.2 upgrade was released.
+ - Updated WPFUI version to 3.0.0. Thanks [@niels9001](https://github.com/niels9001)!
+ - XAML Styler is now fully enabled in the solution. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

Comment:
... But what if a developer doesn't have it installed?

Reply:
Changed it to mention it's when CI runs.

Reply:
I've addressed the feedback in https://github.com/microsoft/PowerToys/pull/31648/commits/b3b0ee3f9c6233f9b79a9dbe0b9397cbab7a5136. Thank you!